framer/Layer.coffee

```diff
@@ -884,16 +884,13 @@ class exports.Layer extends BaseClass
 	##############################################################
 	## ANIMATION
 
-	# Used to animate to a state with a specific name
-	# We lookup the stateName and call 'animate' with the properties of the state
-	animateToState: (stateName, options={}) ->
-		return @states.machine.switchTo(stateName, options)
-
 	animate: (properties, options={}) ->
 
 		# If the properties are a string, we assume it's a state name
 		if _.isString(properties)
-			return @animateToState(properties, options)
+			# Support options as an object
+			options = options.options if options.options?
+			return @states.machine.switchTo(properties, options)
```

Comment:
It would be nicer to put properties in a new variable `stateName` here to make it more clear what is happening